hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
What's the static kubectl downside?  Can't get the current user and can't
use libc DNS?

On Tue, Sep 29, 2015 at 10:27 PM, Andy Goldstein <notifications@github.com>
wrote:

> In hack/build-cross.sh
> <https://github.com/kubernetes/kubernetes/pull/11694#discussion_r40724385>
> :
>
> > @@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
> >  kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
> >
> >  KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
> > +KUBE_STATIC_OVERRIDES=("kubectl")
>
> @smarterclayton <https://github.com/smarterclayton> @mikedanese
> <https://github.com/mikedanese> is this ok? We need a static kubectl so
> we can run it in a container as part of the e2e tests for exec + http proxy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/11694/files#r40724385>.
>